transformers/integration_tests/plugin_test.go

```diff
@@ -167,6 +176,7 @@ var _ = Describe("Plugin test", func() {
 			goPath, soPath, pathErr = eventConfig.GetPluginPaths()
 			Expect(pathErr).ToNot(HaveOccurred())
 			g, initErr = p2.NewGenerator(eventConfig, dbConfig)
+			g.SetMigrationManager(&StubMigrationManager{})
```

Comment:
It would try to run migrations - but would determine they were finished. This doesn't work now since in test the public schema isn't technically migrated, just loaded.

 I tried for way too long to get this to play nice via various tricks, before I just realized I didn't want this in tests anyway.

Reply:
yeah I mean nice catch, tests running migrations is ... :donotwant: